docs/docs/faq/question_05.md

```diff
@@ -0,0 +1,46 @@
+
+# How should I save and load objects in `sbi`?
+
+`NeuralPosterior` objects are picklable.
+```python
+import pickle
+
+# ... run inference
+posterior = inference.build_posterior()
+
+with open("my_path.pkl", "wb"):
+    pickle.dump(posterior)
+```
+
+`NeuralInference` objects are not picklable. There are two workarounds:
+
+- Pickle with `dill` (has to be installed with `pip install dill` first)
+```python
+import dill as pickle
```

Comment:
```suggestion
import dill
```
Anything can cause quite some mayhem with larger scripts. I suggest to convey intend clearly.